### PR TITLE
README: make it clear rustc is the compiler

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

**Comment by nagisa** [comment]
I think current text makes it pretty clear rustc is, in fact, the Rust compiler.

**Comment by tshepang** [comment]
@nagisa the listing made it look like *The Rust compiler* and *rustc* could be different things, so this PR makes it more clear.

**Comment by liigo** [comment]
Yes, it was already clear.

**Comment by tshepang** [comment]
@nikomatsakis only to those who are already aware that rustc is *The Rust compiler*

**Comment by nagisa** [comment]
@tshepang current sentence correctly uses an [appositive](https://en.wikipedia.org/wiki/Apposition) which

> identifies or renames another word in a sentence--most often a noun that immediately precedes it.
 -- [about.com](http://grammar.about.com/od/basicsentencegrammar/a/buildappositive.htm)

**Comment by tshepang** [comment]
@nagisa I am aware of that usage, and that the English used here is valid. To prove it, here's another alternative that would also make things less ambiguous:

> This repo contains the code for `rustc`, the Rust compiler. It also contains standard libraries, tools and documentation for Rust.

That's less ambiguous than the current:

> This repo contains the code for `rustc`, the Rust compiler, as well  as standard libraries, tools and documentation for Rust.

**Comment by soltanmm** [comment]
Given that we've all seen typos, poor grammar, and the like peppering various code-related documents, there is at least one possibility of ambiguity in the original sentence. This possible ambiguity arises between parsing the sentence as having poor grammar, and parsing the sentence as exactly written. This change uses English language features such as the parenthetical statement to significantly decrease the possibility of that ambiguity arising for readers skimming the text. I think this is a pragmatic change from that perspective.

**Comment by tshepang** [comment]
Support finally! Thanks @soltanmm.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Rust is a systems programming language that is fast, memory safe and
 multithreaded, but does not employ a garbage collector or otherwise
 impose significant runtime overhead.
 
-This repo contains the code for `rustc`, the Rust compiler, as well
+This repo contains the code for the compiler (`rustc`), as well
```
Comment (bluss) [inline README.md]: What if we just remove the comma separation, and write it like this? *This repo contains the code for the Rust compiler `rustc`, as well as standard libraries, tools and documentation for Rust.*

Comment (tshepang) [inline README.md]: I think the English with your suggestion is not valid, given the meaning intended.

**Pull request merged**